server/src/workers/api.ts

```diff
@@ -14,9 +14,18 @@ import { useSwagger } from 'src/utils/misc';
 
 const host = process.env.HOST;
 
+function parseTrustedProxy(input?: string) {
+  if (input === undefined || input === '') {
```

Comment:
```suggestion
  if (!input) {
```

Reply:
Ok, you prefere a less explicit check :D

Reply:
I think it's equally explicit :P
If you prefer keeping it I'm also fine with that though

Reply:
The end result is the same, I think that the explicit check is easier to see as not everyone knows that empty string is false.
I've already applied the change anyway 😄